### PR TITLE
Create Biweekly_meetings_with_Arielle.md

### PR DESCRIPTION
**Pull request by pherterich**
Kicking off our notes - @Arielle-Bennett does this look okay to you?

### CODE REVIEW

**Review by Arielle-Bennett — APPROVED** [review]

**Comment by Arielle-Bennett** [comment]
I think it's great although I don't currently have write access @KirstieJane so can't merge this...

**Comment by pherterich** [comment]
@Arielle-Bennett are you okay with the updates so far? Then we can merge and going forward we're finally on track for the bi-weekly updates 🤞🏻

**Pull request merged**
